### ISSUE TITLE
Cache resized images for faster build times

### ISSUE THREAD

**Issue by whmountains** [herrstucki/responsive-loader#37]
After installing this my webpack built times have gone up to 5+ seconds making my feedback look much longer.  Would it be possible for this plugin to cache the created images and only re-compress them if the parameters were different?

**Comment by nicobrinkkemper** [comment]
Could something like this help? https://github.com/kentcdodds/babel-plugin-preval
It would mean that the responsive loader works synchronously once and then just fall back to file-loader.

**Comment by herrstucki** [comment]
Hi, I just released `v1.0.0` which gives you the option to use [sharp](https://github.com/lovell/sharp). That should significantly speed up the build.

In addition, when you use the `disable` option, the images are not processed but just passed through which should give you very little overhead in development. (If you can't upgrade to webpack 2+ yet: the option exists in v0.7.0 too, there it's called `pass`).

@nicobrinkkemper that's a Babel plugin and won't do us much good here. Also, AFAIK Babel plugins can only run synchronously which is a major drawback compared to webpack (since multiple images can't be processed in parallel).
